CHANGES.rst

```diff
@@ -10,6 +10,8 @@ New Features
 
 - ``astropy.convolution``
 
+  - Major change in convolution behavior and keyword arguments (#5782)
```

Comment:
Also maybe mention that un-normalized kernels are now possible in a separate bullet?  That's a distinct new feature, really...